_events/2016-01-31-feminism-is-for-everybody.md

```diff
@@ -0,0 +1,9 @@
+---
+identifier: feminism-is-for-everybody
+slug: feminism-is-for-everybody
+title: "January Book Club"
+date: "2017-01-31 14:00:00"
```

Comment:
I think @lizhubertz is right? if there's a date, then no time. Time: `datetime`.